### DISCUSSION TITLE
Invalid preview and links for code search

### DISCUSSION THREAD

**Discussion by mweibel** [community/community#29640] [Other Feature Feedback, Questions, & Ideas] [answered]
recently the code search seems to be more broken than before. 
For example:
https://github.com/containerd/containerd/blob/a04268132eaf764103b8d4d0672d19da6d88cbd7/pkg/cri/util/id.go

If you click on `GenerateId` then in the context popup -> `References` you'll see a bunch of results, all within the file `pkg/cri/util/id.go`. This is obviously wrong. That file has 29 lines, none of them are anywhere in that file. Also when clicking on any of the results it just updates the hash in the same location. 

![image](https://user-images.githubusercontent.com/307427/184831680-40377c26-1c6f-4bbd-8fc5-91c928848598.png)

**Comment by hendrikvanantwerpen** [comment, marked as answer]
Thank you for bringing this to our attention. We just deployed a fix which should resolve this issue.

**Reply by mweibel** [reply]
thanks! That fix seems to have worked :+1:
